packages/core/src/components/text-input/text-input.stories.js

```diff
@@ -109,3 +109,18 @@ export const Required = () => `
      ${getHelperText()}
     </div>
 `;
+
+export const Number = () => `
```

Comment:
Is it ok for this number input to be with the text-input stories? The only difference is type="number" for the core version.

Reply:
The only benefit of having this in its own category is that it would match the component listing and probably improve findability a bit. Now it can be difficult to find this if the user does not expect it to be under a TextInput category.

Reply:
- [ ] Add category for number input to the core package